core/src/main/java/org/apache/iceberg/hadoop/HadoopCatalog.java

```diff
@@ -59,9 +60,12 @@
  */
 public class HadoopCatalog extends BaseMetastoreCatalog implements Closeable {
   private static final String ICEBERG_HADOOP_WAREHOUSE_BASE = "iceberg/warehouse";
+  private static final String TABLE_METADATA_FILE_EXTENSION = ".metadata.json";
+  private static final PathFilter TABLE_FILTER = path -> path.getName().endsWith(TABLE_METADATA_FILE_EXTENSION);
   private final Configuration conf;
   private String warehouseLocation;
 
+
```

Comment:
Nit: Please remove additional blank lines like this.

Reply:
ok removed